spec/compiler/interpreter/primitives_spec.cr

```diff
@@ -270,20 +270,22 @@ describe Crystal::Repl::Interpreter do
         interpret("23.8_f32.to_{{target_type}}!").should eq(f.to_{{target_type}}!)
       end
 
-      it "interprets Float32#to_{{target_type}}! (negative)" do
-        f = -23.8_f32
-        interpret("-23.8_f32.to_{{target_type}}!").should eq(f.to_{{target_type}}!)
-      end
-
       it "interprets Float64#to_{{target_type}}! (positive)" do
         f = 23.8_f64
         interpret("23.8_f64.to_{{target_type}}!").should eq(f.to_{{target_type}}!)
       end
 
-      it "interprets Float64#to_{{target_type}}! (negative)" do
-        f = -23.8_f64
-        interpret("-23.8_f64.to_{{target_type}}!").should eq(f.to_{{target_type}}!)
-      end
+      {% unless target_type.starts_with?("u") %}
```

Comment:
```suggestion
      {% unless target_type.starts_with?("u") %} # Do not test undefined behavior that might differ (#13736)
```

Reply:
@HertzDevil Do you agree with amending this suggestion?